Patch/json bigquery compatibility (#160)
fivetran/dbt_fivetran_log: CHANGELOG.md

# dbt_fivetran_log v2.2.2

> [PR #158](https://github.com/fivetran/dbt_fivetran_log/pull/158) includes the following updates:

Author: ```suggestion
[PR #160](https://github.com/fivetran/dbt_fivetran_log/pull/160) includes the following updates:
```

## Under the Hood
- Added BigQuery JSON field support for the following model and columns:

>   - `stg_fivetran_platform__log`: `message_data` column. This resulted in a new cte named `field_conversion` to apply the conversion prior to the additional light transformations using the `message_data` column.

Reviewer: ```suggestion
  - `stg_fivetran_platform__log`: `message_data` column. 
    - Added the `field_conversion` cte to apply the conversion to `message_data` prior to additional transformations.
```
